packages/grafana-data/src/utils/OptionsUIBuilders.ts

```diff
@@ -143,6 +143,16 @@ export class PanelOptionsEditorBuilder<TOptions> extends OptionsUIRegistryBuilde
     });
   }
 
+  addStringArray<TSettings>(
```

Comment:
Haven't looked at the code yet - but I have doubts if we want this to be public API. This seems like quite a niche use case. Maybe you could just just `addCustomEditor` for world ping?

Reply:
string[] niche?!   for field config sure... but isn't this for options in general?

Reply:
@ryantxu well, I've used wrong expression :) I was just wondering if we want to provide an UI fur this option, don't know how wildly used it is. But I'm totally fine with adding it- above is just wondering :)